api-guidelines/global/core-principles/quality-standards.md

```diff
@@ -46,30 +54,27 @@ We are conservative in exposing our data and the [Principle of least privilege](
 In addition, we strive to include only the least amount of data necessary to respond to any API call and secure our applications against the [OWASP Top 10 Threats](https://owasp.org/www-project-top-ten/).
 
 We also restrict the rate limit to specific resources to prevent misuse.
-::::
 
-:::: accordion Performance
+## Performance
+
 We identify and analyze key metrics for different groups of interest.
 The bandwidth of possible metrics ranges from purely technical information such as uptime, latencies, and error rates to business insights such as SDK, version adoption, as well as Time to First Hello World (TTFHW), or API usage growth.
-::::
 
-:::: accordion Documentation & Support
-We help both vendors during development and users of our API with the integration by offering suitable ways of exchange and support.
-As the primary resource for explaining our API and its capabilities, documentation must be as accessible to the audience as possible.
-We provide all consumers of our API with comprehensive, professional, up-to-date, and complete information.
-::::
+## Documentation and support
 
-:::: accordion Communication
+We help both vendors during development and users of the API with the integration by offering suitable ways of exchange and support.
+As the primary resource for explaining the API and its capabilities, documentation must be as accessible to the audience as possible.
+We provide all consumers of the API with comprehensive, professional, up-to-date, and complete information.
 
-We always keep both developers and consumers of our API informed through appropriate channels.
+## Communication
+
+We always keep both developers and consumers of the API informed through appropriate channels.
 Changes and [deprecations](../../rest/compatibility/README.md#deprecation-of-http-apis) are communicated regularly and actively.
 Therefore, we establish different synchronous and asynchronous communication channels to support developers and consumers.
-::::
 
-:::: accordion Developer Experience
-API consumers should have fun using our API.
+## Developer experience
+
+API consumers should have fun using the API.
 Our goal is to provide seamless experience to developers when writing software, and to increase their efficiency.
-API consumers should be comfortable using our API in their programming language of choice, finding the functionality they need, as well as using the output.
-We give developers the right tools to help them succeed and aim to provide an as short as possible TTFHW.
-::::
-:::::
+API consumers should be comfortable using the API in their programming language of choice, finding the functionality they need, as well as using the output.
+We give developers the right tools to help them succeed and aim to provide an as short as possible TTFHW.
```

Comment:
I had to look up TTFHW... usually we do not want to use abbreviations without introducing them first... Is it such a well-known concept?

Reply:
I also did not know what it meant :sweat_smile:.